ocaml/xapi/xapi_vm.ml

```diff
@@ -60,6 +60,14 @@ let assert_can_boot_here ~__context ~self ~host =
   assert_can_boot_here ~__context ~self ~host ~snapshot ~do_cpuid_check:true ()
 
 let retrieve_wlb_recommendations ~__context ~vm =
+  let string_of_float f =
```

Comment:
Can you call this `compat_string_of_float`, that makes it clearer when reading the place it gets called that this is not the standard one.